addons_marketplace/marketplace-manager/services/marketplace_service.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import threading
+
+import docker
+from db import marketplace_db
+
+
+def verify_addon(addon_id, addon):
+    logging.info(f"verifying addon-{addon_id}...")
+    client = docker.from_env()
+    for service in addon["services"]:
+        image = service.get("image")
+        image_id = None
+        try:
+            # TODO utilize opencontainers image spec to verify image instead of docker
+            logging.info(f"Pulling image: {image}")
+            pulled_image = client.images.pull(image)
```

Comment:
If I'm not mistaken from the docker python SDK there is a command method `get_registry_data` that gives you the metadata from the image registry given an image name. No image pull requested. Maybe we can use that for this check?

Reply:
Oh I see, that's even better. Thanks a lot :D